docs/source/tutorials/projects.rst

```diff
@@ -0,0 +1,1213 @@
+
+.. code-block::
+
+    from encord.client import EncordClient
+
+    # Initialize project client
+    client = EncordClient.initialise(
+      "<project_id>",  # Project ID
+      "<project_api_key>"  # API key with model.inference access
+    )
+
+    # Run inference and print inference result
+    inference_result = client.model_inference(
+      "<model_iteration_id>",  # Model iteration ID
+      data_hashes=['video1_data_hash', 'video2_data_hash'],  # List of data_hash values for videos/image groups
+      detection_frame_range=[0, 100],  # Run detection on frames 0 to 100
+    )
+    print(inference_result)
+
+
+You can run inference on existing videos/image groups in the platform.
+You can do the same by specifying the ``data_hashes`` parameter which is the list of unique identifiers of the video/image groups on which you want to run inference.
+You can define confidence, intersection-over-union (IoU) and polygon coarseness thresholds.
+The default confidence threshold is set to ``0.6``, the default IoU threshold is set to ``0.3`` and the default value for the polygon coarseness is set to ``0.005``.
+
+
+.. code-block::
+
+    inference_result = client.model_inference(
+      "<model_iteration_id>",  # Model iteration ID
+      data_hashes=['video1_data_hash', 'video2_data_hash'],  # List of data_hash values for videos/image groups
+      detection_frame_range=[0, 100],  # Run detection on frames 0 to 100
+      conf_thresh=0.6,  # Set confidence threshold to 0.6
+      iou_thresh=0.3,  # Set IoU threshold to 0.3
+      rdp_thresh=0.005,  # Set polygon coarseness to 0.005
+    )
+    print(inference_result)
+
+
+The model inference API also accepts a list of locally stored images to run inference on.
+In case of locally stored images only JPEG and PNG file types are supported for running inference.
+
+.. code-block::
+
+    inference_result = client.model_inference(
+      "<model_iteration_id>",  # Model iteration ID
+      file_paths=['path/to/file/1.jpg', 'path/to/file/2.jpg'],  # Local file paths to images
+      detection_frame_range=[1,1],
+    )
+    print(inference_result)
+
+For running inference on locally stored videos, only MP4 and WebM video types are supported.
+
+
+.. code-block::
+
+    inference_result = client.model_inference(
+      "<model_iteration_id>",  # Model iteration ID
+      file_paths=['path/to/file/1.mp4', 'path/to/file/2.mp4'],  # Local file paths to videos
+      detection_frame_range=[0, 100],  # Run detection on frames 0 to 100
+    )
+    print(inference_result)
+
+
+The model inference API also accepts a list of base64 encoded strings.
+
+.. code-block::
+
+    inference_result = client.model_inference(
+      "<model_iteration_id>",  # Model iteration ID
+      base64_strings=[base64_str_1, base_64_str_2],  # Base 64 encoded strings of images/videos
+      detection_frame_range=[1,1],
+    )
+    print(inference_result)
+
+Limits on the input values
```

Comment:
This bullet list is not rendering properly.

Replace - with : to match our general style in other docs please.